src/otx/algo/classification/mmconfigs/multiclass_classification/dino_v2.yaml

```diff
@@ -0,0 +1,6 @@
+backbone:
+  name: dinov2_vits14_reg
+  frozen: false
+head:
+  in_channels: 384
+  num_classes: 1000
```

Comment:
Wouldn't it be better to have this as an argument to the model? DINO doesn't actually use mm, so it seems a bit odd to have configs moved to mmconfigs.

Reply:
It is currently implemented as MM-style and it must be revisited in the future. The reason for this change is because I encountered annoying test errors since `jsonargparse` kept failing to parse something in `DictConfig`. What you mentioned will be amortized in the future revisit for this model to disassemble the MM-style.